README.rst

```diff
@@ -0,0 +1,109 @@
+wenda_gpu: fast domain adaptation for genomic data
+==================================================
+
+Overview
+--------
+
+Domain adaptation allows for development of predictive models even in
+cases with limited or unlabeled sample data, by developing predictors
+for the data of interest (target data) using labeled data from a similar
+distribution (source data). This repo is a fast implementation of one
+domain adaptation method, weighted elastic net domain adaptation, or
+wenda. It leverages the complex interactions between biological features
+(such as genes) to optimize a model’s predictive power on both source
+and target datasets.
+
+Installation
+------------
+
+This package can be installed using pip:
+
+::
+
+   pip install wenda_gpu
+
+Alternatively, you can install the latest development version directly
+from this GitHub repository:
+
+::
+
+   pip install git+https://github.com/greenelab/wenda_gpu
+
+Usage
+-----
+
+The most basic usage of wenda is this:
+
+::
+
+   from wenda_gpu import wenda_gpu as wg
+
+   source_data, target_data = wg.load_data(prefix="sample")
+   source_data_normed, target_data_normed = wg.normalize_data(source_data, target_data)
+   wg.train_feature_models(source_data_normed, target_data_normed, prefix="sample")
+   source_y = wg.load_labels(prefix="sample")
+   wg.train_elastic_net(source_data_normed, source_y, target_data_normed, prefix="sample")
+
+For a step-by-step tutorial in running wenda_gpu, consult
+(https://github.com/greenelab/wenda_gpu/example/wenda_gpu_quick_usage.ipynb).
+
+Directory structure
+-------------------
+
+By default, wenda_gpu creates the following structure in your working
+directory:
+
+::
+
+   working_directory
+       ├── data
+       │   └── prefix
+       │       ├── source_data.tsv
```

Comment:
Is the source_data.tsv and target_data.tsv something that wenda creates or an input that the user needs to create? Based on the instructions, I'd assume that all these files are created by wenda

Reply:
Good catch. Those ones a user will have to create. I'll rewrite the section.